include/pfs/drp/stella/SpectraTemplates.hpp

```diff
@@ -503,37 +503,86 @@ bool pfs::drp::stella::Spectrum<SpectrumT, MaskT, VarianceT, WavelengthT>::ident
 
     ndarray::Array< double, 1, 1 > D_A1_FittedWLenCheck = math::getSubArray( D_A1_WLen, I_A1_IndexCheckPos );
 
+    std::vector<string> S_A1_Args(3);
```

Comment:
I'm confused as to why all this code is in a header file: is that really necessary?